### ISSUE TITLE
Warning and failed messages when running eclipse in nosim mode

### ISSUE THREAD

**Issue by mortalisk** [equinor/ert#4650]

**Comment by oysteoh** [comment]
Our action on this should be just to verify that the error message from loading summary files which is not there  makes sense.

**Comment by oysteoh** [comment]
Could this one be solved with the new parser @mortalisk ?

**Comment by oysteoh** [comment]
Could be related to #5055

**Comment by oysteoh** [comment]
@eivindjahren  do you have any input to this issue ? Something we could / should fix?

**Comment by eivindjahren** [comment]
The current model of ert is that there are `FORWARD_MODEL`s which work as a black box. Ert doesn't know what data they are responsible for producing, what data they require etc. 

This is probably the most convoluted example, where the presence of `INSERT_NOSIM` causes the `ECL` forward model to not produce the summary data required by the `SUMMARY` keyword. We could possibly create some heuristic "plugin validators", ie. the NOSIM plugin could warn the user when the `SUMMARY` keyword is present...

**Comment by oysteoh** [comment]
Hmm... Actually i'm not sure how simple this is actually... I think i remember seeing a case where they injected `INSERT_NOSIM` then ran some stuff not requiring `SUMMARY` .. then further on in the config-file `REMOVE_NOSIM` was injected and then running of stuff requiring a `SUMMARY` ...

**Comment by eivindjahren** [comment]
Good point! We definitely will not be able to confidently say that no summary is generated, so the solution is currently limited to a warning. Not sure how helpful that is.

**Comment by oysteoh** [comment]
I think this issue is ready to be closed and then if needed created again at some point ......
